index.js

```diff
@@ -317,6 +318,7 @@ async function update(vm, workItem) {
 async function comment(vm, workItem) {
 	let patchDocument = [];
 
+	console.log(`CommentText: ${vm.comment_text}`)
```

Comment:
ditto

Reply:
Yeah, until we know we have comments synced.